plugins/commands/reporter/src/main/kotlin/ReporterCommand.kt

```diff
@@ -158,6 +158,12 @@ class ReporterCommand(descriptor: PluginDescriptor = ReporterCommandFactory.desc
         .default(ortConfigDirectory.resolve(ORT_LICENSE_CLASSIFICATIONS_FILENAME))
         .configurationGroup()
 
+    private val licenseCategoriesToInclude by option(
+        "--license-categories-to-include",
+        help = "The licenses of the packages have to be categorized with one of these categories, specified as " +
+            "comma-separated values."
+    ).split(",")
```

Comment:
This should only be an option of the reporter command if it is planned that all reporters (or at least the majority) support it. But I don't think that this is the case, because reports like the `WebAppReport` should always show the complete data.
Also, this PR only implements this for the `CtrlxReporter` so I would suggest to make it a report specific option and reconsider making it a global option if it is actually needed by more reports.

Reply:
Did you see [this](https://github.com/oss-review-toolkit/ort/pull/9842#discussion_r1930684464) discussion, @mnonnenmacher?